### DISCUSSION TITLE
[Feature] ability to hide individual photos

### DISCUSSION THREAD

**Discussion by benjamindaines** [immich-app/immich#7528] [Feature Request]
### The feature

In order to be able to replace the default gallery app on the phone, immich must have the ability to hide photos, which seems to be lacking. If I'm lucky enough to have my Monday morning blessed with some naughty photos sent to me, really cannot have them just hanging out there in the app if I open it for some kind of work-related purpose. 

Should be an option along with archive and delete when selecting an image or album. 

There would also need to be a mechanism of temporarily showing hidden images. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by schuhbacca** [comment]
Duplicate of: https://github.com/immich-app/immich/discussions/2451

**Comment by alextran1502** [comment]
YOu can archive those images so they won't show up on your timeline.
